lib/SQL/Translator/Diff.pm

```diff
@@ -92,7 +92,7 @@ sub schema_diff {
     $options ||= {};
 
     my $obj = SQL::Translator::Diff->new( {
-      %$options,
+      sqlt_args => $options,
```

Comment:
hey, why did you change this line?
It looks to me like the way it's meant to be used is to passthrough args to SQLT::Diff, one arg of which is the args which are passed down to the inner SQLT instance

Reply:
At this point, I am not quite sure why I did that. I have been making all of these changes as a result of trying to get a project off the ground using DBIx::Class with an Oracle DB and either DBIx::Class::Schema::Versioned or DBIx::Class::DeploymentHandler for deployment. I know at one point I was running into an issue where schema_diff was getting called directly by a module and when that was happening the sqlt_args that I had passed to the module were getting lost. Changing that line fixed the issue but I hadn't realized how many other things it actually broke. I reverted that and everything in my code seems to be working properly so I am guessing that was related to an issue I was having with DBIx::Class::DeploymentHandler. If I run into the issue again I will do more thorough testing and open a new issue.

Reply:
k, great